tests/filecheck/mlir-conversion/with-mlir/parser-printer/attribute_names.mlir

```diff
@@ -0,0 +1,50 @@
+// RUN: xdsl-opt %s | filecheck %s
```

Comment:
```suggestion
// RUN: XDSL_ROUNDTRIP
```

Reply:
Occured to me that this might not actually do the same thing. Perhaps ignore this suggestion